From 248e87869d748cfb72d3a5d3b2b91ad178172e30 Mon Sep 17 00:00:00 2001
From: Vasileios Anagnostopoulos <anagnwstopoulos@hotmail.com>
Date: Tue, 9 May 2023 12:48:29 +0200
Subject: [PATCH] Add the pkgconfigdir location

This change is necessary, because while building asio in yocto the
compilation is failing and is complaining because the variable is
undefined

Signed-off-by: Vasileios Anagnostopoulos <anagnwstopoulos@hotmail.com>
Upstream-Status: Submitted [https://github.com/chriskohlhoff/asio/pull/1294]
---
 asio/Makefile.am | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Makefile.am b/Makefile.am
index b7760851..bc4f75b3 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -1,6 +1,7 @@
 AUTOMAKE_OPTIONS = foreign dist-bzip2 dist-zip
 
 pkgconfig_DATA = asio.pc
+pkgconfigdir = $(libdir)/pkgconfig
 
 SUBDIRS = include src
 
-- 
2.34.1

